### ISSUE TITLE
Multilingual pages

### ISSUE THREAD

**Issue by kertase** [sethmacleod/dimension#6]
Hi,

I am not sure if this is really an issue. But I think that in Multilingual mode when a page is intro.md it should appear in all languages unless there is specifically intro.xx.md for the specific language available.

This would make it possible to reduce redundancy when creating pages that are the same for all languages.

Thanks.

L.

**Comment by sethmacleod** [comment]
Thanks for the tip. I'd like to get this in line with Hugo best practices. I'll have to reread the docs.

**Comment by sethmacleod** [comment]
I've reread the docs on multilingual mode, and I believe I'm doing it correctly. Can you rephrase?

It seems like files only appear in their respective language code. So, elements.md only appears in the English version of the site, and german.de.md only appears in the German version of the site. These are files without equivalents in the other language.

**Comment by kertase** [comment]
I had read :
If a string does not have a translation for the current language, Hugo will use the value from the default language. If no default value is set, an empty string will be shown.

But it seems to be for strings and not for pages.

Sorry for making the trouble.

L.
